### ISSUE TITLE
Loki Explore - Log Visualization Timespan doesn't honor the timerange provided

### ISSUE THREAD

**Issue by agung-kargo** [grafana/grafana#26808]
**What happened**:
Searching log on Grafana Explore with datasource Loki with timerange **Last 3 hours**, however the Log visualization only shows less than 3 hour time span.

See gif here: 
![Peek 2020-08-05 19-49](https://user-images.githubusercontent.com/63702999/89414801-359e9a00-d755-11ea-91eb-a55fd6e62b0d.gif)

**What you expected to happen**:
I expect the Log visualisation panel shows the correct time range, honoring the timerange selected at the top menu.

In grafana v6.7.3, the Log visualization panel shows the correct time range (horizontal range) although it only shows the bar limited by the Loki max lines number. But this is tolerable as the user can drill down (select) the timerange via the Log panel in specific time.

With current grafana v7.1.1, since Log visualization panel doesn't honor the selected timerange, it makes harder for us to pinpoint or navigate the log in specific time (except manually changing the timerange with specific `from` and `to`).

**How to reproduce it (as minimally and precisely as possible)**:
See the gif:
![Peek 2020-08-05 19-49](https://user-images.githubusercontent.com/63702999/89414801-359e9a00-d755-11ea-91eb-a55fd6e62b0d.gif)

**Anything else we need to know?**:

**Environment**: 
- Grafana version: v7.1.1
- Data source type & version: docker image `grafana/loki:1.5.0`
- OS Grafana is installed on: EKS
- User OS & Browser: Ubuntu 18.04 & Mozilla Firefox
- Grafana plugins: -
- Others: -

**Comment by torkelo** [comment]
I think this is by design, the time range in the graph will zoom on on the lines returned. But there is no info on what is happening so I think there is something off here.

**Comment by agung-kargo** [comment]
> I think this is by design, the time range in the graph will zoom on on the lines returned. 

@torkelo  I see, thanks for confirming this Loki graph behavior. Is there any consideration to make the graph show the timerange set on the timerange top panel? Just like how previous grafana v6 does. See this screenshot (grafana version 6.5.0 using the same Loki data source), where regardless of number of lines returned, it honors the selected timerange.

The screenshot below is querying Loki log in the last 1 hour. The graph timerange is 1 hour, but only show bar line limited to 1000 lines only (which is expected as well).

![image](https://user-images.githubusercontent.com/63702999/89880929-070a3e80-dbef-11ea-9bff-bf58a41d565b.png)

As you can see in the sreenshot above, this is so to make it consistent with the graph behavior of other data source like Prometheus. Common use case in Explore mode is to have Split panel between `Metrics`(prometheus data source) and `Log` (Loki data source). Having the same graph showing the same time range will help people navigating on certain time range (and zoom in via mouse selection in the graph)

> But there is no info on what is happening so I think there is something off here.

Basically what I was trying to say in the description is: there is a change behavior from grafana v6 vs v7.1.1 and whether it's bug or not.

**Comment by davkal** [comment]
Thanks for the feedback. We chose the current approach to make sure that users did not think their logs are missing. A nicer approach would be to use the correct time range, and show a hint that a limit was reached. This will be worked on soon.

**Comment by davkal** [comment]
Added graph/panel label because we need support of showing a range hint as done in #14286

**Comment by mayconritzmann** [comment]
Hello, I am having a similar problem when using the log panel.

The time interval is not respected.

**Comment by xRoker** [comment]
The new behaviour is less convenient for a workflow where you select a larger timerange and try to narrow it down on the graph.

**Comment by pettai** [comment]
Latest Loki/Grafana (docker images) still suffers from this issue. Any ETA yet?

**Comment by ericxw** [comment]
Looking for a solution here as well. Having a time range for the query only to show a small time range based on max lines of logs which is only 5000 is not ideal for digging into potential issues, especially when the off the shelf application is fairly noisy. There's not much on what logs to expect so it's hard to say, just have a query that narrows down on what you're looking for to reduce number lof lines.

Currently, because of this, one application log is only showing 15min worth of range regardless of how wide the range has been set. Example: Selected time range is now-2days, shown logs is last 15min.
